internal/prune/agent/remote.go

```diff
@@ -0,0 +1,50 @@
+/*
+ *  *******************************************************************************
+ *  * Copyright (c) 2019 Edgeworx, Inc.
+ *  *
+ *  * This program and the accompanying materials are made available under the
+ *  * terms of the Eclipse Public License v. 2.0 which is available at
+ *  * http://www.eclipse.org/legal/epl-2.0
+ *  *
+ *  * SPDX-License-Identifier: EPL-2.0
+ *  *******************************************************************************
+ *
+ */
+
+package pruneagent
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/eclipse-iofog/iofogctl/internal"
+	"github.com/eclipse-iofog/iofogctl/internal/config"
+	"github.com/eclipse-iofog/iofogctl/pkg/iofog/install"
+	"github.com/eclipse-iofog/iofogctl/pkg/util"
+)
+
+func (exe executor) remoteAgentPrune(agent config.Agent) error {
+	ctrl, err := internal.NewControllerClient(exe.namespace)
+	// If controller exists, prune the agent
+	if err == nil {
```

Comment:
error is never returned.

replace with
if err != nil {
return err
}

Reply:
updated